plugins/datastore.js

```diff
@@ -41,6 +41,16 @@ const SCHEMA_SCAN = Joi.object().keys({
     timeKey: Joi.string(),
     aggregationField: Joi.string()
 });
+const SCHEMA_QUERY = Joi.object().keys({
+    table,
+    queries: Joi.array().items(
+        Joi.object().keys({
+            dbType: Joi.string().required(),
+            query: Joi.string().required()
+        })).required(),
+    replacements: Joi.object(),
```

Comment:
might want to be define what this object contains here

Reply:
This object's keys would change based on the query, as such I don't think its possible to define what the object contains